helm/chart-operator-chart/templates/rbac.yaml

```diff
@@ -18,6 +18,12 @@ rules:
   - chartconfigs
   verbs:
   - "*"
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - "get"
```

Comment:
The pods access is needed for helmclient to get the TIller pod.

Reply:
<3